setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = clover-energy
+version = 5.0.0a1.post1
+author = Phil Sandwell, Ben Winchester and Hamish Beath
+author_email = philip.sandwell@gmail.com,benedict.winchester@gmail.com,hamishbeath@outlook.com
+description = Continuous Lifetime Optimisation of Variable Electricity Resources
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/phil-sandwell/CLOVER
+project_urls =
+    Bug Tracker = https://github.com/phil-sandwell/CLOVER/issues
+classifiers =
+    Programming Language :: Python :: 3.7
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+install_requires =
```

Comment:
Just a question: is this smart and updates `requirements.txt`(or the other way around)? Or in the future will we need to do it manuall?

Reply:
This is not smart unfortunately and needs to be updated separately.
I think (and I might be wrong) that `install_requires` lists things that, when you install CLOVER from `pypi` using `pip install clover-energy`, python will automatically install to ensure that everything is up-to-date. The `requirements.txt` file is for if you are developing CLOVER and download the code from Github to make sure that you've got the same stuff